feat: Addition DX arguments (#116)
moderneinc/moderne-docs: administrator-documentation/moderne-dx/how-to-guides/configure-dx-organizations.md

* You have deployed Moderne DX in your environment
* You have already [configured Moderne DX to connect to your Artifactory instance](configure-dx-with-artifactory-access.md)

## File based organization structure

> The simplest way to achieve the organization structure is to supply a repos.csv file directly in DX.

Reviewer: ```suggestion
The simplest way to achieve the organization structure is to supply a `repos.csv` file directly in DX.
```

> Downsides of file based organization structure:

Reviewer: ```suggestion
That being said, there are a few downsides of file based organization structure:
```

> - You cannot configure DevCenter

Reviewer: I struggled to parse this a bit at first as I thought there were multiple DevCenters. I think we could fix this by just slightly rephrasing this and adding a link:

```suggestion
- You cannot configure any [DevCenter](/administrator-documentation/moderne-platform/how-to-guides/dev-center.md)
```

To generate this `repos.csv`, we recommend using "[repo fetchers](https://github.com/moderneinc/moderne-organizations/blob/main/repo-fetchers/README.md)" inside our Organizations service template.

## Service based organization structure

> A more feature rich replacement of the file based approach is to run the Organizations service. This allowes you to use additional features like the DevCenter, organization based access control and per organization commit options.

Reviewer: ```suggestion
A more feature rich replacement of the file based approach is to create an [Organizations service](/administrator-documentation/moderne-dx/how-to-guides/configure-dx-org-service.md). This allows you to use additional features like the DevCenter, organization based access control, and per organization commit options.
```

### Organizations service template and API

You have two main options for building this service. You can:

1. (**Recommended**) Fork our [Organizations service template](https://github.com/moderneinc/moderne-organizations) and modify it to meet your needs. Please see the [README](https://github.com/moderneinc/moderne-organizations/blob/main/README.md) for how to spin this up quickly. It can be as simple as updating a CSV file.
2. Build your own service that fulfills the [GraphQL contract](https://github.com/moderneinc/moderne-organizations/blob/main/src/main/resources/schema/moderne-organizations.graphqls) using any GraphQL stack (e.g., NodeJS, Rust, C#, etc.)

We generally recommend forking the template and modifying it as, in most cases, that will be faster and easier than building it yourself. Regardless of which one you choose, however, some developer time will be required on your end.

### Deploying the service

How you deploy the service is largely up to your company. With that being said, there are a few important things to be aware of:

> * Moderne will make a request per repository to the Organizations service once every 10 minutes by default (you can change this interval in your configuration. Please ensure that you have metrics to track how this service is performing so you can adjust it over time.

Reviewer: ```suggestion
* Moderne will make a request per repository to the Organizations service once every 10 minutes by default (you can change this interval in your configuration). Please ensure that you have metrics to track how this service is performing so you can adjust it over time.
```
